# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: security/manager/ssl/nsNSSModule.cpp
# Commit: 2b647eb982bc
# Full Hash: 2b647eb982bc0068d39fe58c6f59bebe55909343
# Author: Dana Keeler <dkeeler@mozilla.com>
# Date: 2023-06-07 21:43:58
# Regressor Bug: 1834862
# File Overlap Count: 1
# Description:
#   Bug 1834862 - implement nsICryptoHash in rust r=jschanck,supply-chain-reviewers
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D179727
# ==============================================================================

diff -r 7c11166969ff -r 2b647eb982bc security/manager/ssl/nsNSSModule.cpp
--- a/security/manager/ssl/nsNSSModule.cpp	Wed Jun 07 15:45:14 2023 +0000
+++ b/security/manager/ssl/nsNSSModule.cpp	Wed Jun 07 15:56:37 2023 +0000
@@ -15,7 +15,6 @@
 #include "mozilla/ModuleUtils.h"
 #include "mozilla/SyncRunnable.h"
 #include "nsCertTree.h"
-#include "nsCryptoHash.h"
 #include "nsNSSCertificateDB.h"
 #include "nsPK11TokenDB.h"
 #include "nsRandomGenerator.h"
@@ -96,7 +95,6 @@
 IMPL(PKCS11ModuleDB, nullptr)
 IMPL(nsNSSCertificateDB, nullptr)
 IMPL(nsCertTree, nullptr)
-IMPL(nsCryptoHash, nullptr, ProcessRestriction::AnyProcess)
 IMPL(ContentSignatureVerifier, nullptr)
 IMPL(nsRandomGenerator, nullptr, ProcessRestriction::AnyProcess)
 IMPL(OSKeyStore, nullptr, ProcessRestriction::ParentProcessOnly,